linera-views/src/rocks_db.rs

```diff
@@ -172,30 +173,77 @@ pub struct RocksDbClient {
 }
 
 impl RocksDbClient {
+    /// testing the existence of table
+    pub async fn test_existence<P: AsRef<Path>>(path: &P) -> Result<bool, RocksDbContextError> {
+        let options = rocksdb::Options::default();
+        let result = DB::open(&options, path);
+        match result {
+            Ok(_) => Ok(true),
+            Err(error) => match error.kind() {
+                rocksdb::ErrorKind::InvalidArgument => Ok(false),
```

Comment:
That's surprising… it's not `ErrorKind::NotFound`?

Reply:
I thought so as well, but no.